js/5etools-monsters.js

```diff
@@ -1339,6 +1348,19 @@ function d20plusMonsters () {
 								}
 							}
 
+							// Load lair actions and regional effects
+							if (data.legendaryGroup) {
+								const legGroup = DataUtil.monster.getMetaGroup(data);
+								if (legGroup) {
+									if (legGroup.lairActions) {
+										renderFluff += renderer.render({entries: legGroup.lairActions, name: "Lair Actions"}, depth = 2);
```

Comment:
- Depth for legendary stuff should be `-1`, i.e. a `"section"`
- `depth = 2` is "not valid JS syntax"--it works, but it works because `depth = 2` creates a global variable `depth` with value 2; that assignment evaluates to `2`; and then the `2` gets passed through to the function below. E.g.:

```js
myFunc = (depth = 123) => console.log(depth, window.depth)
myFunc()
myFunc(depth = 5)
console.log(depth)
```

```suggestion
										renderFluff += renderer.render({entries: legGroup.lairActions, name: "Lair Actions"}, -1);
```